Sources/PT.PM.CSharpParseTreeUst/Converter/CSharpRoslynParseTreeConverter.cs

```diff
@@ -87,7 +87,10 @@ public override Ust Visit(SyntaxNode node)
 
             if (root == null)
             {
-                root = new RootUst(children.FirstOrDefault()?.Root?.SourceCodeFile, CSharp.Language);
```

Comment:
why you decide to remove `SourceCodeFile` from here?

Reply:
I've discovered that it's `null` in any case. Thus there is no sense in such expression.

Reply:
so `RootUst` can include private field `root`, and property can look like `get => root ?? this`